app/controllers/concerns/registrar.rb

```diff
@@ -54,6 +54,14 @@ def passes_invite_reqs
     invitation[:present]
   end
 
+  # Checks if the current number of users is less than the max allowed
+  def new_user_allowed?
+    if !Rails.configuration.max_registered_enabled || User.get_available_count > 0
```

Comment:
In ruby you don't need all the if else. Just a single line is idiomatic. Return is implied
```
def new_user_allowed?
  !Rails.configuration.max_registered_enabled || User.get_available_count > 0
end
```